compiler/include/symbol.h

```diff
@@ -478,6 +478,7 @@ class FnSymbol : public Symbol {
   virtual void    printDocs(std::ostream *file, unsigned int tabs);
 
   void            setThrowsError();
+  bool            getThrowsError();
```

Comment:
I'd consider naming this method throwsError() (and the private field would need a different name in that case).